doc/classes/TextureRect.xml

```diff
@@ -5,6 +5,7 @@
 	</brief_description>
 	<description>
 		Used to draw icons and sprites in a user interface. The texture's placement can be controlled with the [member stretch_mode] property. It can scale, tile, or stay centered inside its bounding rectangle.
+		[b]Note:[/b] You should enable [member flip_v] when using a TextureRect to display a [ViewportTexture]. Otherwise, the image will appear upside down.
```

Comment:
Or you can flip with [Viewport.render_target_v_flip](https://docs.godotengine.org/en/stable/classes/class_viewport.html#class-viewport-property-render-target-v-flip) (which would be useful if the Viewport is used in multiple places)